rojak-ui-app/RojakAndroid/app/src/main/java/id/artificialintelligence/rojak/models/Candidate.java

```diff
@@ -1,18 +1,20 @@
 package id.artificialintelligence.rojak.models;
 
 
+import com.google.gson.annotations.SerializedName;
+
 public class Candidate {
-    private int id;
-    private String fullName;
-    private String aliasName;
-    private String placeOfBirth;
-    private String dateOfBirth;
-    private String religion;
-    private String websiteUrl;
-    private String photoUrl;
-    private String fbpageUsername;
-    private String twitterUsername;
-    private String instagramUsername;
+    @SerializedName("id") private int id;
+    @SerializedName("full_name") private String fullName;
+    @SerializedName("alias_name") private String aliasName;
+    @SerializedName("place_of_birth") private String placeOfBirth;
+    @SerializedName("date_of_birth") private String dateOfBirth;
+    @SerializedName("religion") private String religion;
```

Comment:
Thanks for the feedback, I know it's not needed. However, in my opinion if I add those the code would be more readable since all the fields consistenly have `@SerializedName` property. Well, if other contributors thinks that would be unnecessary then I will delete those.